CMakeLists.txt

```diff
@@ -21,12 +21,35 @@ if(POLICY CMP0077)
 endif()
 
 # Version information
-file(READ ${CMAKE_SOURCE_DIR}/VERSION.txt VERSION_STRING)
-string(REGEX MATCHALL "^([0-9]+)\\.([0-9]+)\\.([0-9]+)" _ ${VERSION_STRING})
+if(DEFINED ENV{VERSION_STRING})
+    set(VERSION_STRING $ENV{VERSION_STRING})
+else()
+  set(VERSION_FILE "${CMAKE_SOURCE_DIR}/VERSION.txt")
+  if (EXISTS "${VERSION_FILE}")
+    file(READ ${VERSION_FILE} VERSION_STRING)
+  else()
+    set(VERSION_STRING "0.0.1")
+    message("Could not parse version file from: ${VERSION_FILE}")
+    message("Defaulting to ${VERSION_STRING}")
+  endif()
+
+  string(REGEX MATCHALL "^([0-9]+)\\.([0-9]+)\\.([0-9]+)" _ ${VERSION_STRING})
+  set(OPENQASM_VERSION_MAJOR ${CMAKE_MATCH_1})
+  set(OPENQASM_VERSION_MINOR ${CMAKE_MATCH_2})
+  set(OPENQASM_VERSION_PATCH ${CMAKE_MATCH_3})
+  set(OPENQASM_VERSION_TRIPLE ${OPENQASM_VERSION_MAJOR}.${OPENQASM_VERSION_MINOR}.${OPENQASM_VERSION_PATCH})
```

Comment:
Don't you want to pull this out of the if-else block so that it executed regardless of which path sets the `VERSION_STRING` var?

Reply:
Actually, you already have this in lines 46-50. Isn't that sufficient?

Reply:
Good catch 👍 , I think this was left over from a merge or something. Removed in 455e678